Formula/tracker.rb

```diff
@@ -20,6 +20,7 @@ class Tracker < Formula
     sha256 x86_64_linux:   "2b4c9a6c6afffba09852f9b003622ed275d1739455e515c507d73ce7b29a8fd1"
   end
 
+  depends_on "glib-utils" => :build
   depends_on "gobject-introspection" => :build
   depends_on "meson" => :build
```

Comment:
Looks like we need explicit Python3 for Linux build even though they are indirectly installed in build environment (`python@3.10` via `meson`/`glib-utils`, `python@3.9` via `gobject-introspection`).

May just use `depends_on "python@3.10" => :build` 

Could use `uses_from_macos "python" => :build, since: "catalina"` but `python@3.10` is already in build environment so might as well use that.

Reply:
I'm thinking that less dependencies on python are better in this case.